mongodbatlas/resource_mongodbatlas_project.go

```diff
@@ -78,9 +78,10 @@ func resourceMongoDBAtlasProject() *schema.Resource {
 				Default:  true,
 			},
 			"api_keys": {
-				Type:     schema.TypeSet,
-				Optional: true,
-				Computed: true,
+				Type:       schema.TypeSet,
+				Optional:   true,
+				Computed:   true,
+				Deprecated: "This parameter is deprecated,please use project API Key resource instead",
```

Comment:
Could you add this message to the `provider.go` (similar to  [provider.go#L42](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/b4045e1c5f98432a1127495e8ef238fd092ab513/mongodbatlas/provider.go#L42))?
It would be great to have only one deprecation message that we can use in all the resources/data sources. Thanks!

Reply:
Added deprecation message from provider.go